distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Extensions.enso

```diff
@@ -303,6 +304,23 @@ Text.match self pattern=".*" case_sensitivity=Case_Sensitivity.Sensitive =
     compiled_pattern = Regex.compile pattern case_insensitive=case_insensitive
     compiled_pattern.matches self
 
+## Compile the text into a regex pattern that can be used for matching.
+
+   Arguments
+   - case_insensitive: Enables or disables case-insensitive matching. Case
+     insensitive matching behaves as if it normalises the case of all input
+     text before matching on it.
+
+   If an empty regex is used, `compile` throws an Illegal_Argument error.
```

Comment:
```suggestion
   If an empty regex is used, `compile` throws an `Illegal_Argument` error.
```

We really need to come up with some style guide. But for the time being, I think we need to at laest be consistent - if `compile` is in backticks the error type should be too.

Reply:
Done